src/Umbraco.Web.UI.Client/src/packages/tiptap/components/input-tiptap/tiptap-toolbar.element.ts

```diff
@@ -108,7 +108,7 @@ export class UmbTiptapToolbarElement extends UmbLitElement {
 			flex-direction: column;
 
 			position: sticky;
-			top: -25px;
+			top: var(--umb-tiptap-top,-25px);
```

Comment:
I think you should document this CSS Property in the JSDocs for this Class. We use that to highlight CSS Props for customizations and as well then to know if it could be a need for extensions.

Reply:
@nielslyngsoe, I've updated the class and added JSDocs as requested, let me know if you think that something should be adjustend.